posts/2023/09/26/new-ansible-galaxy.md

```diff
@@ -0,0 +1,100 @@
+
+We already have a URL called old-galaxy.ansible.com pointing to a read-only version of the galaxy.ansible.com service as it is today.
+
+On September 30, we will switch the IP records used by galaxy.ansible.com to use the records that beta-galaxy.ansible.com uses. 
+
+* Summary: https://old-galaxy.ansible.com/ will point to old Galaxy
+* https://galaxy.ansible.com/ will point to new [galaxy_ng](https://github.com/ansible/galaxy_ng/)
+* https://beta-galaxy.ansible.com/ will point to new [galaxy_ng](https://github.com/ansible/galaxy_ng/)
+
+This means if the switch is not successful, we can easily switch back or ask the community to try the older service. But we have tested a lot, so everything should go to plan.
+
+ 
+
+## After the Switch
+@rochacbruno has written up this awesome [table](https://github.com/ansible/galaxy_ng/discussions/1729) to show what was in Galaxy vs the new beta Galaxy site. You will notice not everything has been reimplemented into [galaxy_ng](https://github.com/ansible/galaxy_ng/) due to time constraints, but we are open to discussions on [forum.ansible.com](https://forum.ansible.com) if something should come back, and more importantly, we wish for the community to help design new features to be introduced. 
+
+A good example of features that have not yet been reformulated is the current scoring system.We are retaining the old scores, so nobody loses out on the kudos they have earned already, however, we feel that a new scoring system needs to be implemented,and we’d love to begin soliciting community contributions for how we can improve it. 
+
+| Feature                 | Galaxy  | Beta Galaxy | Reason    |
+|-------------------------|-------- |-------------|-----------|
+| Sign in/up via GitHub   | ✔️     | ✔️         |           |
+| API Key                 | ✔️     | ✔️         |           |
+| E-mail/App Notifications | ✔️     | ❌          | Low usage |
+| Following authors and content | ✔️ | ❌        | Low usage  |
+| Multiple e-mail address per account | ✔️ | ❌ | Low usage |
+| Search with filters |✔️ | ✔️ | Improved on Beta |
+| Download Count | ✔️ | ✔️ |  |
+| Popular Tags | ✔️ | TBD | |
+| Platform mark | ✔️ | ❌ | Feature replaced by tags |
+| Content Score | ✔️ | TBD | Read only data will be kept, <br> scoring system will be redesigned |
+| Content list and search | ✔️ | ✔️ | Improved on beta |
+| Content docs | ✔️ | ✔️ | Improved on beta |
+| Upload new collections| ✔️ | ✔️ | Improved on beta |
+| Import logs | ✔️ | ✔️ | Improved on beta |
+| List my namespaces | ✔️ | ✔️ | Simplified on beta |
+| Admin namespace access for contributors | ✔️ |✔️ | Simplified on beta |
+| Add multiple provider namespaces for collections | ✔️ | ❌ | Was not being used |
+| UI to Import Role from Github | ✔️ | ❌ | Recommendation is to publish collections instead, <br> existing roles can still be maintainer via CLI |
+| Task Management | ❌ | ✔️ | User can watch tasks spawned,<br> useful for watching status of CLI imports |
+|
```

Comment:
Can we remove this last pipe character or does that affect table rendering?

Reply:
Yes, I tried removing it and found out it affect the table rendering